src/main/java/flaxbeard/immersivepetroleum/common/blocks/tileentities/HydrotreaterTileEntity.java

```diff
@@ -212,6 +212,7 @@ public void onProcessFinish(MultiblockProcess<SulfurRecoveryRecipe> process){
 	
 	@Override
 	public void tick(){
+		super.tick();
 		checkForNeedlessTicking();
```

Comment:
Would need to be removed because it is already being called first by the `super.tick()` now.

Reply:
ok, I can make that change with no issue, I didn't know I had to remove it, but fair point I can go ahead and do that change.